From c7153027dfa58352e9c429f140d9ef798d673992 Mon Sep 17 00:00:00 2001
Message-ID: <c7153027dfa58352e9c429f140d9ef798d673992.1699403993.git.sharlatanus@gmail.com>
In-Reply-To: <0117df90b1ef43a67d4fa86e5b58e60f7c0a52e5.1699403992.git.sharlatanus@gmail.com>
References: <0117df90b1ef43a67d4fa86e5b58e60f7c0a52e5.1699403992.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 29 Oct 2023 22:58:25 +0000
Subject: [PATCH 59/67] gnu: go-github-com-xeipuuv-gojsonreference: Move to
 (gnu packages golang-web).

* gnu/packages/golang.scm (go-github-com-xeipuuv-gojsonreference): Move from
here...
* gnu/packages/golang-web.scm: ...to here.

Change-Id: Iba8903cba4efa0b806cb19237db3ee51a46646c6
---
 gnu/packages/golang-web.scm | 27 +++++++++++++++++++++++++++
 gnu/packages/golang.scm     | 26 --------------------------
 2 files changed, 27 insertions(+), 26 deletions(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index 1f9fe0ffca..7f6ced049c 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -1227,6 +1227,33 @@ (define-public go-github-com-xeipuuv-gojsonpointer
 programming language.")
       (license license:asl2.0))))
 
+(define-public go-github-com-xeipuuv-gojsonreference
+  (let ((commit "bd5ef7bd5415a7ac448318e64f11a24cd21e594b")
+        (revision "0"))
+    (package
+      (name "go-github-com-xeipuuv-gojsonreference")
+      (version (git-version "0.0.0" revision commit))
+      (source
+       (origin
+         (method git-fetch)
+         (uri (git-reference
+               (url "https://github.com/xeipuuv/gojsonreference")
+               (commit commit)))
+         (file-name (git-file-name name version))
+         (sha256
+          (base32 "1xby79padc7bmyb8rfbad8wfnfdzpnh51b1n8c0kibch0kwc1db5"))))
+      (build-system go-build-system)
+      (arguments
+       '(#:import-path "github.com/xeipuuv/gojsonreference"))
+      (propagated-inputs
+       (list go-github-com-xeipuuv-gojsonpointer))
+      (home-page "https://github.com/xeipuuv/gojsonreference")
+      (synopsis "Implementation of JSON Reference for Go")
+      (description
+       "This package provides an implementation of JSON Reference for the Go
+programming language.")
+      (license license:asl2.0))))
+
 (define-public go-github-com-xeipuuv-gojsonschema
   (let ((commit "6b67b3fab74d992bd07f72550006ab2c6907c416")
         (revision "0"))
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index fb01dd4679..cf2c0dcfc0 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -12125,32 +12125,6 @@ (define-public go-github-com-go-test-deep
 when comparing complex types like structures and maps.")
     (license license:expat)))
 
-(define-public go-github-com-xeipuuv-gojsonreference
-  (let ((commit "bd5ef7bd5415a7ac448318e64f11a24cd21e594b")
-        (revision "0"))
-    (package
-      (name "go-github-com-xeipuuv-gojsonreference")
-      (version (git-version "0.0.0" revision commit))
-      (source (origin
-                (method git-fetch)
-                (uri (git-reference
-                      (url "https://github.com/xeipuuv/gojsonreference")
-                      (commit commit)))
-                (file-name (git-file-name name version))
-                (sha256
-                 (base32
-                  "1xby79padc7bmyb8rfbad8wfnfdzpnh51b1n8c0kibch0kwc1db5"))))
-      (build-system go-build-system)
-      (arguments
-       '(#:import-path "github.com/xeipuuv/gojsonreference"))
-      (propagated-inputs (list go-github-com-xeipuuv-gojsonpointer))
-      (home-page "https://github.com/xeipuuv/gojsonreference")
-      (synopsis "Implementation of JSON Reference for Go")
-      (description
-       "This package provides an implementation of JSON Reference for the Go
-programming language.")
-      (license license:asl2.0))))
-
 (define-public go-github-com-niemeyer-pretty
   (package
     (name "go-github-com-niemeyer-pretty")
-- 
2.41.0

